roles/check-swlib/tasks/main.yml

```diff
@@ -16,13 +16,27 @@
 - name: check-swlib | Validate base software release
   shell: |
     set -o pipefail
-    {% for i in item.files %}
-      gsutil ls -L gs://{{ swlib_mount_src }}/{{ i }} | \
-        awk '/md5/ { print $3 }'
+    {% for f in item.files %}
+    if gsutil ls gs://{{ swlib_mount_src }}/{{ f.name }} >/dev/null 2>&1; then
+      f_name="{{ f.name }}"
+      md5sum="{{ f.md5sum }}"
+    else
+      f_name="{{ f.alt_name | default('x') }}"
+      md5sum="{{ f.alt_md5sum | default('x') }}"
+    fi
+    actual_md5=$(gsutil ls -L gs://{{ swlib_mount_src }}/${f_name} | awk '/md5/ { print $3 }')
+    if [ $? -ne 0 ] ; then
```

Comment:
`[[` and `]]` as per https://google.github.io/styleguide/shellguide.html#s6.3-tests
Actually, given the complexity of the logic in this shell snippet, can we even run sample generated shell through a shell linter?